index.html

```diff
@@ -22,7 +22,7 @@
     <meta name="twitter:description" content="locize - Bridging the gap between translation and development. With locize we remove the pain in the translation process. No more delays in shipping your software because of missing translations. Translator could keep up with changes from day one. The continuous localization process keeps up with your demanding business. Stop waiting - start localizing.">
     <meta name="twitter:creator" content="@locize">
     <meta name="slack-app-id" content="A71NM84Q6">
-    <title>locize - continuous localization (by the creators of i18next)</title>
+    <title>locize - the pay-per-use software localization management platform</title>
```

Comment:
@shdigitalpy said:

the main slogan would be mentioned in h3 but this could not be changed as this is i18next script.

h3 suggestion was

Bridging the gap between translation and development with locize, a modern and affordable localization management platform. It makes your website, app, game or whatever your project is, global, vibrant, and more engaging especially when unleashing the continuous localization capabilities.

h1 suggestion: h1 suggestion: localization management platform for continuous localization
h1 localization management platform - Bridging the gap between translation and development. (not possible to SEO as it's too long max 600px long)

We need main keyword in title otherwise the site would not rank for localization management platform.

Reply:
@jamuhl ok for me, just wanted to make sure you see this